jest.config.js

```diff
@@ -3,4 +3,5 @@ module.exports = {
   moduleNameMapper: {
     '^@/(.*)': '<rootDir>/src/$1',
   },
+  testPathIgnorePatterns: ['_mockdata.'],
```

Comment:
When moving the `*_mockdata` files to `__fixtures__` you do not need this line any more.